server/test/stubs/mock-data.report.js

```diff
@@ -0,0 +1,34 @@
+export default {
+  validReport: {
+    comment: 'This article plagiarises another article that can be found here https://onezero.medium.com/when-the-web-loses-its-memory-1c0fc4375ba9',
+    typeId: 'fc73ad52-1b04-4cba-a54f-e88b74555472',
+    articleId: 'efbd2ccd-4e06-4ecb-bfe0-baf303cd5577'
+  },
+  invalidReport: [
+    {
+      comment: '',
+      typeId: 'fc73ad52-1b04-4cba-a54f-e88b74555472',
+      articleId: 'efbd2ccd-4e06-4ecb-bfe0-baf303cd5577'
+    },
+    {
+      comment: 'This article plagiarises another article that can be found here https://onezero.medium.com/when-the-web-loses-its-memory-1c0fc4375ba9',
+      typeId: '',
```

Comment:
Why not use our authors haven site instead of medium? That would give it a more original feel. What do you think?

Reply:
@TheDrizzyWay nice one 😄 I will do just that.